source/docs/software/labview-dashboard/troubleshooting-dashboard-connectivity.rst

```diff
@@ -36,13 +36,27 @@ Recognizing SmartDashboard Connectivity
 
 The typical way to recognize connectivity with the Java SmartDashboard is to add a Connection Indicator widget and to make sure your code is writing at least one key during initialization or disabled to trigger the connection indicator. The connection indicator can be moved or re-sized if the Editable checkbox is checked.
```

Comment:
This widget doesn't exist for Glass

Reply:
I don't understand what you mean.  There are 3 sections for glass, smartdashboard, and shuffleboard.  The section you are pointing out is for SmartDashboard so it shouldn't apply to glass